torchtitan/experiments/deepseek_v3/moe_kernels.py

```diff
@@ -147,6 +147,7 @@ def generate_permute_indices(
     alignment: int,
     use_cpu: bool = False,
     block_size: int = 128,
+    max_blocks: int = 1024,
```

Comment:
nit: no need to expose these two knobs to the user-facing API maybe?

Reply:
yes I can remove.  I was exposing that mostly so we could do a bit of perf tuning on both llama4 and DeepSeek but doesn't need to be in the user api.

Reply:
removed